diff --git a/node_modules/@chakra-ui/nuxt/lib/plugin.js b/node_modules/@chakra-ui/nuxt/lib/plugin.js
index afe037f..a5b8897 100644
--- a/node_modules/@chakra-ui/nuxt/lib/plugin.js
+++ b/node_modules/@chakra-ui/nuxt/lib/plugin.js
@@ -3,6 +3,12 @@ import { createClientDirective } from '@chakra-ui/vue/src/directives'
 
 const theme = <%= JSON.stringify(options.theme, null, 2) %>
 
+const breakpointSizes = ['sm', 'md', 'lg', 'xl'];
+
+theme.breakpoints.forEach((breakpoint, index) => {
+  theme.breakpoints[breakpointSizes[index]] = breakpoint;
+});
+
 Vue.prototype.$chakra = {
   theme,
   icons: <%= JSON.stringify(options.icons, null, 2) %>
@@ -17,5 +23,9 @@ if (process.client) {
 
   // VScrollLock
   const VScrollLock = require('v-scroll-lock').default
-  Vue.use(VScrollLock)
+  Vue.use(VScrollLock, {
+    bodyScrollOptions: {
+      reserveScrollBarGap: true,
+    }
+  })
 } 
